oauthlib/oauth1/rfc5849/endpoints/signature_only.py

```diff
@@ -34,13 +34,17 @@ def validate_request(self, uri, http_method='GET',
         """
         try:
             request = self._create_request(uri, http_method, body, headers)
-        except errors.OAuth1Error:
+        except errors.OAuth1Error as e:
```

Comment:
Yeah, I don't love that one, either. But let sleeping dogs lie.

Thanks for changing it.